docs/getting-started.md

```diff
@@ -22,6 +22,36 @@ Now you can use the `redis-cli` to run the kvrocks server as Redis on port `6666
 redis-cli -p 6666
 ```
 
+For persisted data, prefer using a docker volume over a bind-mounted volume to avoid file permission or performance problems. Eg:
```

Comment:
I like that. The section header belongs a bit lower though. Without a mounted volume, all data is lost when the container is stopped. Since mounting one is absolutely required, it makes sense to provide an example that minimizes pitfalls. The example intentionally moves the data dir because the default kvrocks.conf location conflicts with the default data dir location. (It's too late to change the entrypoint, eh?)